src/Tests/CoreTests/Output/OutputBuilderTest.cs

```diff
@@ -70,5 +72,104 @@ public void TestReplace()
             
             Assert.AreEqual(correctedMessage, parsedMessage);
         }
+        
+        [Test]
+        public void TestReplaceWithNullNewStr()
+        {
+            var replaceWord = "big brown";
+            var correctedMessage = "The  dog.";
+            
+            var output = new OutputBuilder().Append(originalMessage);
+            output.Replace(replaceWord, null);
+            var parsedMessage = output.Parse(terminalOptions);
+            Assert.AreEqual( correctedMessage, parsedMessage );
+        }
+                
+        [Test]
+        public void TestReplaceWithNullOldStr()
+        {
+            var output = new OutputBuilder().Append(originalMessage);
+            output.Replace(null, null);
+            var parsedMessage = output.Parse(terminalOptions);
+            Assert.AreEqual( originalMessage, parsedMessage );
+        }
+
+        [Test]
+        public void TestAppend_IntOverload()
+        {
+            var testData = new List<Tuple<string, int>>()
```

Comment:
nit: This looks like it works.  Since we don't have a lot of unit tests yet, I wonder if there's more we can do to clean this up even more to set up our best standards for following, even if it doesn't matter much here.  (I didn't have any unit testing when I took on championing WheelMUD, but I have a lot more now, though not much still in C#.)  I feel we can get each test case to be their own [Test] and thus get the test results to accurately report all the passes and failures as separate cases, making it easier to diagnose failures and the like.  And maybe with a pattern that's even easier to follow...  Consider maybe:
```
        [Test]
        public void TestAppendStringAndIntMinValue()
        {
            TestAppendStringAndInt("Edge negative: ", int.MinValue, "Edge negative: -2147483648");
        }

        [Test]
        public void TestAppendStringAndIntMaxValue()
        {
            TestAppendStringAndInt("Edge positive: ", int.MaxValue, "Edge positive: 2147483647");
        }

        ...

        private void TestAppendStringAndInt(string s, int i, string expectedResult)
        {
            var result = new OutputBuilder().Append(s).Append(i).Parse(terminalOptions);
            Assert.AreEqual(result, expectedResult);
        }
```

Reply:
I agree with about tests and i can modify them, but i will take some time for me. Im little bit run out of time. Ill do my best